clowdapp.yaml

```diff
@@ -0,0 +1,69 @@
+---
+apiVersion: v1
+kind: Template
+metadata:
+  name: floorist
+objects:
+- apiVersion: cloud.redhat.com/v1alpha1
+  kind: ClowdApp
+  metadata:
+    name: floorist
+  spec:
+    database:
+      sharedDbAppName: ${SHARED_DB_APP_NAME}
+    objectStore:
+      - ${BUCKET_NAME}
```

Comment:
As discussed, we can't have the prod bucket provided via clowder.

Reply:
I've added a configuration system on #10  to allow to force read the bucket name from the environment even on Clowder managed deployments, that should work. if it doesn't I think we can create a second template that doesn't have the objectStore definition.

Reply:
:+1: